app/Policies/OrganizationPolicy.php

```diff
@@ -6,14 +6,26 @@
 use App\Models\User;
 use Illuminate\Auth\Access\HandlesAuthorization;
 use Illuminate\Auth\Access\Response;
+use Illuminate\Support\Str;
 
 class OrganizationPolicy
 {
     use HandlesAuthorization;
 
-    public function before(User $user): null|bool
+    public function before(User $user, string $ability): null|Response
     {
-        return $user->isAdministrator() ? true : null;
+        if ($user->isAdministrator() && $ability === 'viewAny') {
+            return Response::allow();
+        }
```

Comment:
As this only appears to be needed for the `viewAny` ability, I think it could be moved there.